client/cypress/integration/visualizations/counter_spec.js

```diff
@@ -0,0 +1,191 @@
+/* global cy, Cypress */
+
+import { createQuery } from '../../support/redash-api';
+
+const SQL = `
+  SELECT 27182.8182846 AS a, 20000 AS b, 'lorem' AS c UNION ALL
+  SELECT 31415.9265359 AS a, 40000 AS b, 'ipsum' AS c
+`;
+
+describe('Counter', () => {
+  const viewportWidth = Cypress.config('viewportWidth');
+
+  beforeEach(() => {
+    cy.login();
+    createQuery({ query: SQL }).then(({ id }) => {
+      cy.visit(`queries/${id}/source`);
+      cy.getByTestId('ExecuteButton').click();
+    });
+  });
+
+  it('creates simple Counter', () => {
+    cy.clickThrough(`
+      NewVisualization
+      VisualizationType
+      VisualizationType.COUNTER
+
+      Counter.General.ValueColumn
+      Counter.General.ValueColumn.a
+    `);
+
+    cy.wait(200); // eslint-disable-line cypress/no-unnecessary-waiting
```

Comment:
What's the wait for?

Reply:
There were some issues similar to other visualizations tests - visualization needed some delay to properly update itself before taking a snapshot.

Reply:
Then perhaps it makes more sense to place it before the snapshot (instead of before the element assertion) 👆

Reply:
@ranbena Done